f2/src/locales/en.json

```diff
@@ -278,16 +278,13 @@
   "landingPage.reportingOptions3": "If you encountered the incident online, you can also try reporting it to the website or app where it happened.",
   "landingPage.title": "Report a scam or computer crime",
   "landingPage.warning": "Reporting does not always lead to an investigation and we likely won’t be able to recover what was lost or damaged.",
-  "locationInfoForm.Warning": "Enter a valid Canadian postal code in the format shown or leave this blank.",
```

Comment:
Since an error warning exists we should use this instead of the hard coded value.